parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -19,6 +17,17 @@ installDeps() {
     if [[ $OS_VERSION == "2.0" ]]; then
       systemctl --now mask nftables.service || exit $ERR_SYSTEMCTL_MASK_FAIL
     fi
+
+    # Install the package repo for the specific OS version.
+    # AzureLinux 3.0 uses the azurelinux-repos-cloud-native repo
+    # Other OS, e.g., Mariner 2.0 uses the mariner-repos-cloud-native repo
+    if [[ $OS_VERSION == "3.0" ]]; then
+      echo "Installing azurelinux-repos-cloud-native"
```

Comment:
May  I ask how these repos are maintained? Are we planning to migrate all binaries to PMC in the future?

Reply:
There are 3 binaries still hosting in acs-mirror. The plan is to move these all to PMC and use Dalec to manage/build them. Paul's team is working on CNI plugin-ins. Another one to be moved is Azure-cni. This is for Linux only though. Not sure about the plan of Windows.